tests/test_measures.py

```diff
@@ -116,15 +115,6 @@ def circuit():
     @pytest.mark.parametrize(
         "cases",
         [
-            [
```

Comment:
Thanks @chaeyeunpark! I'm just wondering how this hadn't been caught by `cpp-tests` before :thinking: